config/sidekiq.yml

```diff
@@ -0,0 +1,5 @@
+:concurrency: 4
+:queues:
+  - default
+  - mailers
```

Comment:
ActionMailer places mail jobs on a queue named `mailers` by default, which is why we need to specify this here